<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ex√©cution courante</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>331</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_53">
         <property name="text">
          <string>Device de la trace</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <item>
          <widget class="QLineEdit" name="browseEdit_exec">
           <property name="text">
            <string>/dev/ttyACM0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browse_button_exec">
           <property name="text">
            <string>Parcourir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="readDevice_button">
           <property name="text">
            <string>Lire</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="signal_display_frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="ignition4_layout_2">
         <item>
          <widget class="QLabel" name="ignition4_title_label_exec">
           <property name="text">
            <string>Bougie 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition4_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="ignition3_layout_2">
         <item>
          <widget class="QLabel" name="ignition3_title_label_exec">
           <property name="text">
            <string>Bougie 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition3_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="ignition6_layout_2">
         <item>
          <widget class="QLabel" name="ignition6_title_label_exec">
           <property name="text">
            <string>Bougie 6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition6_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="ignition2_layout_2">
         <item>
          <widget class="QLabel" name="ignition2_title_label_exec">
           <property name="text">
            <string>Bougie 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition2_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="ignition5_layout_2">
         <item>
          <widget class="QLabel" name="ignition5_title_label_exec">
           <property name="text">
            <string>Bougie 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition5_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="ignition1_layout_2">
         <item>
          <widget class="QLabel" name="ignition1_title_label_exec">
           <property name="text">
            <string>Bougie 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition1_value_label_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="engine_rpm_exec">
           <property name="text">
            <string>rpm moteur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="engine_rpm_value_exec">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>331</width>
        <height>116</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="execution_layout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_54">
             <property name="text">
              <string>Fichier d'enregistrement de la trace</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_30">
             <item>
              <widget class="QLineEdit" name="browseEdit_save_exec">
               <property name="text">
                <string>/home/Kappa</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browse_button_save_exec">
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="save_button_exec">
             <property name="text">
              <string>Commencer l'enregistrement</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="chart_tabWidget_exec">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>721</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ignition chart</string>
       </attribute>
       <widget class="QtCharts::QChartView" name="ignition_chart_exec">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Engine RPM chart</string>
       </attribute>
       <widget class="QtCharts::QChartView" name="engine_chart_exec">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Lecture de trace</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>291</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="trace_layout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="select_trace_label">
         <property name="text">
          <string>S√©lectionner une trace </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="trace_browse_layout">
         <item>
          <widget class="QLineEdit" name="browseEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browse_button">
           <property name="text">
            <string>Parcourir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>291</width>
        <height>122</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="execution_layout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="execution_button">
           <property name="text">
            <string>Ex√©cuter la trace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pause_button">
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="execution_rotation_layout">
         <item row="0" column="0">
          <widget class="QLabel" name="rotation_freq_label">
           <property name="text">
            <string>Nombre de tours moteur affich√©s par seconde</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="rotation_freq_counter">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="signal_display_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>331</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="ignition4_layout">
         <item>
          <widget class="QLabel" name="ignition4_title_label">
           <property name="text">
            <string>Bougie 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition4_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="ignition3_layout">
         <item>
          <widget class="QLabel" name="ignition3_title_label">
           <property name="text">
            <string>Bougie 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition3_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="ignition6_layout">
         <item>
          <widget class="QLabel" name="ignition6_title_label">
           <property name="text">
            <string>Bougie 6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition6_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="ignition2_layout">
         <item>
          <widget class="QLabel" name="ignition2_title_label">
           <property name="text">
            <string>Bougie 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition2_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="ignition5_layout">
         <item>
          <widget class="QLabel" name="ignition5_title_label">
           <property name="text">
            <string>Bougie 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition5_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="ignition1_layout">
         <item>
          <widget class="QLabel" name="ignition1_title_label">
           <property name="text">
            <string>Bougie 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ignition1_value_label">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>rpm moteur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="engine_rpm_value_trace">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="chart_tabWidget_trace">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>721</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ignition chart</string>
       </attribute>
       <widget class="QtCharts::QChartView" name="ignition_chart_trace">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>681</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Engine RPM chart</string>
       </attribute>
       <widget class="QtCharts::QChartView" name="engine_chart_trace">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="engineCycle_display">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>331</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
